ArduCopter/mode_guided.cpp

```diff
@@ -436,29 +522,31 @@ void ModeGuided::pos_control_run()
     // set motors to full range
     motors->set_desired_spool_state(AP_Motors::DesiredSpoolState::THROTTLE_UNLIMITED);
 
-    // run waypoint controller
-    copter.failsafe_terrain_set_status(wp_nav->update_wpnav());
+    // send position and velocity targets to position controller
+    guided_accel_target_cmss.zero();
```

Comment:
I think we can safely remove these two lines to init the accel and vel targets.  I can understand the desire to be safe but there are lots of variables floating around that initialising them when not needed is a waste of flash and makes the code more complex.

Reply:
I sort of disagree there. This makes the code internally consistent with the position controller.